micro_sam/sam_annotator/image_series_annotator.py

```diff
@@ -82,6 +88,7 @@ def image_series_annotator(
             This will take more time when precomputing embeddings, but will then make
             automatic mask generation much faster.
         checkpoint_path: Path to a custom checkpoint from which to load the SAM model.
+        is_volumetric: Whether to use the 3d annotator.
```

Comment:
Add this as an argument for the CLI as well. (See the `main` function at the end of this file).

Reply:
Thanks for spotting these. I updated them all!